common/app/views/support/package.scala

```diff
@@ -184,7 +184,7 @@ object RenderOtherStatus {
   def apply(result: Result)(implicit request: RequestHeader, context: ApplicationContext) = result.header.status match {
     case 404 => NoCache(NotFound)
     case 410 if request.isJson => Cached(60)(JsonComponent(gonePage, "status" -> "GONE"))
```

Comment:
I does return a `200`. I am not sure if that is a problem or not actually. For take downs in AMP it shouldn't make a difference. But potentially we want it to return a 410 just so it is more correct. What do you think @jfsoul ?